### ISSUE TITLE
Hardware / Installation

### ISSUE THREAD

**Issue by guywire** [kikiisdead/minichord#2]
Can you add a hardware used and installation instructions guide to the readme?

**Comment by kikiisdead** [comment]
bro this is the wrong repository again

**Comment by guywire** [comment]
No, I wanted to know how you're making a mini chord. Ibrealize it's a
different repository.

**Comment by kikiisdead** [comment]
It's teensy 4.1 based. I'm basically attaching a microcontroller and several breakout boards to a PCB that I designed. I coded it in the Arduino IDE which is also how I compile and send code to the microcontroller.

The breakout boards I used are the Adafruit CAP1188 for the capacitive touch and a ripoff Adafruit SSD1306 screen. 

[https://youtube.com/shorts/4UmdJhHuOzo](https://youtube.com/shorts/4UmdJhHuOzo) Here is a video of the minichord. 

I designed the PCB myself in KiCad which is free and very easy to use. 

It definitely isn't an elegant solution: it's bulky and not very space efficient. It would be much simpler to have everything on the PCB itself but this is an intro the digital electronics class so we didn't get there yet.
